server/package.json

```diff
@@ -6,7 +6,7 @@
         "start": "ts-node-dev --respawn src/index.ts",
         "start:local": "DB_LOCAL=true ts-node-dev --respawn src/index.ts",
         "start:localwindows": "set DB_LOCAL=true && ts-node-dev --respawn src/index.ts",
-        "dev": "nodemon -L --exec ts-node src/index.ts",
+        "dev": "DB_LOCAL=true nodemon -L --exec ts-node-dev --respawn src/index.ts",
```

Comment:
I could be wrong but I have a feeling this might break the production build since it's also using `npm run dev` as the start script. Is it possible to have a separate dev environment in the dockerfile or docker-compose to handle the local scenario?

Reply:
For sure! Thanks for pointing that out!